src/commands/environments/create.ts

```diff
@@ -28,6 +29,11 @@ export default class EnvironmentCreate extends BaseCommand {
       description: 'Environment Description',
       sensitive: false,
     }),
+    strict: booleanString({
```

Comment:
This was going to be a hidden flag for now, right?

Reply:
Set flag to hidden